scrapy/core/engine.py

```diff
@@ -15,7 +15,8 @@
 
 from scrapy import signals
 from scrapy.core.scraper import Scraper
-from scrapy.exceptions import DontCloseSpider, ScrapyDeprecationWarning
+from scrapy.exceptions import DontCloseSpider, ScrapyDeprecationWarning, \
+    CloseSpider
```

Comment:
:lipstick:  We don’t really use \ at the end if it can be avoided.

```suggestion
from scrapy.exceptions import (
    CloseSpider,
    DontCloseSpider, 
    ScrapyDeprecationWarning,
)   
```